canvas.py

```diff
@@ -0,0 +1,72 @@
+import http.client
+import json
+import logging
+import time
+
+__author__ = 'Islam Elnabarawy'
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Yeah, I went ahead and took it out. I think it would only go into effect if this module was loaded as the main module, but since we don't have any main code in there it's better if we remove it.